ios/FluentUI.Demo/FluentUI.Demo/Demos/ShimmerLinesViewDemoController.swift

```diff
@@ -5,11 +5,61 @@
 
 import UIKit
 
-class ShimmerLinesViewDemoController: DemoController {
-    override func viewDidLoad() {
-        super.viewDidLoad()
+class ShimmerViewDemoController: DemoController {
+	override func viewDidLoad() {
+		super.viewDidLoad()
```

Comment:
Is there a linter we're supposed to run? Looks like it used to be tabs but your settings have it set to spaces

Reply:
We have SwiftLint on iOS that runs if it's installed. I do get lots of lint warnings. However, I don't think we ever specificied a SwiftLint formatting file, and just use the defaults. 

Furthermore, I'm pretty sure most engineers don't have SwiftLint installed AKA it never runs

Reply:
Proper linting on FUA is still an active work item IMO, we should use Swift-Lint by apple like what @cy255 did earlier